package.json

```diff
@@ -49,6 +49,7 @@
     "express-handlebars": "3.0.0",
     "fs-extra": "2.1.2",
     "irc-framework": "2.8.0",
+    "jquery-textcomplete": "1.8.0",
```

Comment:
Good catch, yes!
It doesn't need to be a dependency since it gets built into `lounge.vendor.js` by `npm run build` at release time.

@yashsriv, I'm on the spottiest connection ever these days, would you mind making that change?